dandiapi/api/management/commands/create_dev_dandiset.py

```diff
@@ -6,28 +6,52 @@
 from django.contrib.auth.models import User
 from django.core.files.uploadedfile import SimpleUploadedFile
 import djclick as click
+import faker
 
 from dandiapi.api.models import AssetBlob
 from dandiapi.api.services.asset import add_asset_to_version
 from dandiapi.api.services.dandiset import create_dandiset
 from dandiapi.api.services.metadata import validate_asset_metadata, validate_version_metadata
+from dandiapi.api.services.permissions.dandiset import add_dandiset_owner
 from dandiapi.api.tasks import calculate_sha256
 
 
 @click.command()
 @click.option('--name', default='Development Dandiset')
 @click.option('--owner', 'email', required=True, help='The email address of the owner')
-def create_dev_dandiset(*, name: str, email: str):
+@click.option(
+    '--num-extra-owners',
+    'num_extra_owners',
+    required=False,
+    type=int,
+    default=0,
+    show_default=True,
+    help='The number of extra owners to programmatically generate',
+)
```

Comment:
Should we remove this? Or at least move it to its own management command if you feel its useful to keep? I think there's value in keeping `create_dev_dandiset` as simple as possible, since it's a key part of getting a dev environment set up without having to upload data with the CLI.

Reply:
This was previously a separate management command included in this PR, and I merged it into this command, in order to simplify things. I don't know how much value there is in keeping it. I don't think it warrants its own management command, since at that point it's essentially just adding owners to a dev dandiset. If you don't think it's valuable (or if it adds negative value) we can remove it from here.

My two cents is that it does provide value, since often times when developing locally, we have just a single dandiset that has just one owner (yourself). So having ways to extend that to provide a more realistic test dandiset is useful, I think. Although it does increase the complexity of the management command slightly.

Reply:
I don't have a strong opinion either way. I suppose I lean slightly towards removing it because I don't see myself using it. But, if you think you will use it / it could be useful in general for devs, then I'm not opposed. It does add some complexity, but imo it's so little additional complexity that someone finding it useful is a compelling enough reason to keep it. 

Although I think we should move `Faker` into `dev` dependencies, and only import it inside that conditional. Just to avoid pulling in a large dependency like that into production when it's not really needed

Reply:
Fixed in [1472be2](https://github.com/dandi/dandi-archive/pull/2272/commits/1472be2cd644beede5612908163456a4ac643c43)